docs/trulens/gh_top_intro.md

```diff
@@ -59,7 +59,7 @@ Install the trulens-eval pip package from PyPI.
 pip install trulens-eval
```

Comment:
Do we still need this here? after removing `trulens_explain` we shouldn't need to have this in 2 places